config/config.yaml

```diff
@@ -0,0 +1,12 @@
+project_code: recover
+latest_version: v0.1
```

Comment:
I don't think we'll need the `latest_version` property for recover. Remove it for now.